src/client/ocpp/json/JsonRestChargingStationClient.ts

```diff
@@ -88,13 +91,17 @@ export default class JsonRestChargingStationClient extends ChargingStationClient
 
   private async openConnection(): Promise<unknown> {
     // Log
-    Logging.logInfo({
-      tenantID: this.tenantID,
-      source: this.chargingStation.id,
-      action: ServerAction.WS_REST_CLIENT_CONNECTION_OPENED,
-      module: MODULE_NAME, method: 'onOpen',
-      message: `Try to connect to '${this.serverURL}' ${Configuration.isCloudFoundry() ? ', CF Instance \'' + this.chargingStation.cfApplicationIDAndInstanceIndex + '\'' : ''}`
-    });
+    if (this.dbLogging) {
+      Logging.logInfo({
```

Comment:
In the end why protecting this with a variable as I want to see these logs all the time?

Reply:
In the WSClient code, it's a requirement: the client is used outside the server with no DB connection to the DB.

Reply:
> Only in the WSClient code, it's a requirement: the client is used outside the server with no DB.

For what scenario? Unit Test?

Reply:
Yes to simulate charging station OCPP command. That's the only place where I will not be able to follow the review comment on that topic. Elsewhere, I will remove the console logging and the condition.